src/sgame/sg_bot_util.cpp

```diff
@@ -1198,15 +1198,20 @@ void BotGetIdealAimLocation( gentity_t *self, botTarget_t target, vec3_t aimLoca
 {
 	//get the position of the target
 	BotGetTargetPos( target, aimLocation );
+	bool target_buildable = BotGetTargetType( target ) == entityType_t::ET_BUILDABLE;
+	//this retrieves the target's species, to aim at weak point:
```

Comment:
```suggestion
	// this uses the target's information and species, to aim at a judicious spot:
```

Reply:
right... considering the fact it's just "apply hard-coded offset" instead of "aim for the point with the less armor which can be reached"